src/api/java/jni/cvcJavaApi.h

```diff
@@ -0,0 +1,24 @@
+#ifndef CVC__JAVA_API_H
+#define CVC__JAVA_API_H
+
+#include <iostream>
```

Comment:
Unused include

Reply:
It was used with std::cout for debugging. But now it is not used.

Reply:
Removed